Parsec TPM Evidence (#1)
veraison/parsectpm: README.md

# parsec-tpm

> A Library to Encode and Decode Parsec TPM Evidence as
> detailed in [attested-tls-poc](github.com/CCC-Attestation/attested-tls-poc)

Reviewer: ```suggestion
A Library to handle Parsec TPM Evidence as
detailed in [attested-tls-poc](https://github.com/CCC-Attestation/attested-tls-poc)
```

It provides following features

> * Encode a Parsec TPM Evidence Collection, containing a Key Attestation and Platform Attestation 

Reviewer: ```suggestion
* Encode a Parsec TPM Evidence Collection object, containing a Key Attestation and Platform Attestation 
```

> information to CBOR and JSON formats

Reviewer: ```suggestion
evidence and metadata to CBOR and JSON formats
```

> * Decode a CBOR and JSON formatted Parsec TPM Evidence into a Parsec TPM Evidence Collection

Reviewer: ```suggestion
* Decode a CBOR and JSON formatted Parsec TPM Evidence into a Parsec TPM Evidence Collection object
```

* Verify the signature on the Key and Platform Attestation elements using the supplied public key

> * Sign the Supplied Key and/or Platform Attestation elements using the supplied key

Reviewer: ```suggestion
* Sign the supplied Key and/or Platform Attestation data using the supplied private key
```
